pkg/rbd/driver.go

```diff
@@ -144,7 +152,7 @@ func (r *Driver) Run(conf *util.Config, cachePersister util.CachePersister) {
 		r.cs = NewControllerServer(r.cd, cachePersister)
 	}
 	if !conf.IsControllerServer && !conf.IsNodeServer {
-		r.ns, err = NewNodeServer(r.cd, conf.Vtype)
+		r.ns, err = NewNodeServer(r.cd, conf.Vtype, topology)
```

Comment:
the topology will be empty here isn't it?

Reply:
Addressed in the latest update. Good catch!